README.md

```diff
@@ -6,29 +6,20 @@
 
 ###### To run locally:
 1. Clone this repository to your local folder
-1. Run in terminal
+2. Run in terminal
 ```
 $ pip install -r requirements/dev.txt
 ```
-1. Add `mysite/local_settings.py`, like this:
-```python
-import os
-BASE_DIR = os.path.dirname(os.path.dirname(__file__))
-
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-    }
-}
-
+3. Add `.env` in `mysite` folder, like this:
+```
 DEBUG = True
```

Comment:
No, it shouldn't contain spaces. Better add example file to the repo as I advised you earlier.